d2lbook/build.py

```diff
@@ -33,23 +36,31 @@ def build():
     parser.add_argument('commands', nargs='+', choices=commands)
     parser.add_argument('--tab', default=None,
                         help='The tab to build, if multi-tab is enabled.')
+    parser.add_argument('--style', default=None,
+                        help='The pdf style. It supports cambridge style (cambridge) and sphinx default (None).')
     args = parser.parse_args(sys.argv[2:])
     config = Config(tab=args.tab)
     builder = Builder(config)
     for cmd in args.commands:
-        getattr(builder, cmd)()
+        if cmd == 'pdf':
+            getattr(builder, cmd)(args.style)
```

Comment:
Can you pass the `style` flag similar to `config = Config(tab=args.tab)`? In this case every member method can access it via `self.config.style` without passing via method arguments.

Reply:
done. style argument is removed.